.gitignore

```diff
@@ -37,4 +37,4 @@ build/
 node_modules
 
 # Vagrant
-/tools/vagrant/.vagrant*
+.vagrant/
```

Comment:
.vagrant/ it should not affect you then
At the contrary if folks have different vagrant files and use cases we want to avoid the specific temporary ".vagrant" directory from being in source control